dvc/remote/ssh/__init__.py

```diff
@@ -38,15 +39,27 @@ def __init__(self, repo, config):
 
         parsed = urlparse(self.url)
         self.host = parsed.hostname
+
+        user_config_file = os.path.expanduser("~/.ssh/config")
+        user_ssh_config = dict()
+        if self.host:
+            if os.path.exists(user_config_file):
```

Comment:
Let's put all of this config parsing logic into a separate method/function and use it.

Reply:
Done